executor/adapter.go

```diff
@@ -711,7 +711,8 @@ var QueryReplacer = strings.NewReplacer("\r", " ", "\n", " ", "\t", " ")
 
 func (a *ExecStmt) logAudit() {
 	sessVars := a.Ctx.GetSessionVars()
-	if sessVars.InRestrictedSQL {
+	enable := atomic.LoadUint32(&config.GetGlobalConfig().Log.EnableAuditLog) > 0
+	if enable && sessVars.InRestrictedSQL {
```

Comment:
If `enable` is false, it will do audit log now?

Reply:
You are right, it's wrong.
And after discussing, this audit log switch has been removed now.